tests/update_workflow.go

```diff
@@ -863,6 +864,162 @@ func (s *FunctionalSuite) TestUpdateWorkflow_RunningWorkflowTask_NewNotEmptySpec
  14 WorkflowExecutionCompleted`, events)
 }
 
+func (s *FunctionalSuite) TestUpdateWorkflow_CompletedWorkflow() {
```

Comment:
Test need to be adjust with #6258.

Reply:
I can fix it once https://github.com/temporalio/temporal/pull/6258 is merged 👍